corehq/sql_proxy_accessors/sql_templates/create_case_notify_trigger.sql

```diff
@@ -0,0 +1,9 @@
+CREATE OR REPLACE FUNCTION case_notify() RETURNS trigger AS $$
+DECLARE
+BEGIN
+  PERFORM pg_notify('{{ channel.name }}', row_to_json(NEW));
+  RETURN NEW;
+END;
+$$ LANGUAGE plproxy;
```

Comment:
yeah it's the other language (pg something something), i was just putting it up to show what it'd ~look like. agreed does make sense to put on each shard